src/pages/cli/auth/admin.mdx

```diff
@@ -12,20 +12,70 @@ For example, the ability to list all users in a Cognito User Pool may provide us
 The Amplify CLI can setup a REST endpoint with secure access to a Lambda function running with limited permissions to the User Pool if you wish to have these capabilities in your application, and you can choose to expose the actions to all users with a valid account or restrict to a specific User Pool Group.
 
 ## Enable Admin Queries
+ 
+<BlockSwitcher>
+<Block name="With New Auth resources">
```

Comment:
the casing here is a bit odd, and we'll work with a singular auth resource. Suggesting Title Case as it is used in other block switchers
```suggestion
<Block name="With a New Auth Resource">
```